cmd/admin-log.go

```diff
@@ -0,0 +1,142 @@
+/*
+ * MinIO Client (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+const logTimeFormat string = "15:04:05 MST 01/02/2006"
+
+var adminLogFlags = []cli.Flag{}
+
+var adminLogCmd = cli.Command{
+	Name:            "log",
+	Usage:           "show console logs for minio server",
+	Action:          mainAdminLog,
+	Before:          setGlobalsFromContext,
+	Flags:           append(adminLogFlags, globalFlags...),
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+	  {{.HelpName}} - {{.Usage}}
+  
+  USAGE:
+	  {{.HelpName}} [FLAGS] TARGET
+  
+  FLAGS:
+	  {{range .VisibleFlags}}{{.}}
+	  {{end}}
+  EXAMPLES:
+		  1. Show console logs for a MinIO server with alias 'play'
+				 $ {{.HelpName}} play
+  `,
+}
+
+func checkAdminLogSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 1 {
+		cli.ShowCommandHelpAndExit(ctx, "log", 1) // last argument is exit code
+	}
+}
+
+type logMessage struct {
```

Comment:
Could you add a comment explaining the purpose of this struct? Like it is for adding `JSON` and `String` methods etc.